common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -560,12 +561,11 @@ private long waitForNextTick() {
                 //
                 // See https://github.com/netty/netty/issues/356
                 if (PlatformDependent.isWindows()) {
-                    sleepTimeMs = sleepTimeMs / 10 * 10;
+                    sleepTimeNanos = sleepTimeNanos / 10 * 10;
                 }
 
-                try {
-                    Thread.sleep(sleepTimeMs);
-                } catch (InterruptedException ignored) {
+                LockSupport.parkNanos(sleepTimeNanos);
```

Comment:
I found this tweet: https://twitter.com/mjpt777/status/783017327566749696, which states the absolute minimum sleep on Linux is about 50 microseconds. Unless somebody else have better numbers/references in place, maybe we should mention that lower-bound in docs?

Reply:
@vkostyukov I think I would not mention direct numbers but maybe we could add something like:

```
It highly depends on the system what what the effective minimum tick time will be at the end.
```

Reply:
That sounds good to me 👍